src/taxonomy/taxonomy-menu/TaxonomyMenu.test.jsx

```diff
@@ -26,14 +27,20 @@ const queryClient = new QueryClient();
 const mockSetToastMessage = jest.fn();
 
 const TaxonomyMenuComponent = ({
+  // eslint-disable-next-line react/prop-types
   iconMenu,
-  systemDefined,
-  canChangeTaxonomy,
-  canDeleteTaxonomy,
+  // eslint-disable-next-line react/prop-types
+  systemDefined = false,
+  // eslint-disable-next-line react/prop-types
+  canChangeTaxonomy = true,
+  // eslint-disable-next-line react/prop-types
+  canDeleteTaxonomy = true,
```

Comment:
At this point disabling this is using as many lines of code as simply adding the props? Why not just add the props instead?

Reply:
haha sure, though I did so using TypeScript.